metals/src/main/scala/scala/meta/internal/decorations/SyntheticsDecorationProvider.scala

```diff
@@ -148,21 +148,22 @@ final class SyntheticsDecorationProvider(
       pcHover: Option[l.Hover],
       position: l.Position
   ): Option[l.Hover] = {
-    syntheticsAtLine
-      .find { case (range, _) =>
-        range.getEnd().getCharacter() == position.getCharacter()
-      }
-      .flatMap { case (_, synthetic) =>
-        addToHover(
-          pcHover,
-          "**Synthetics**:\n"
-            +
-              HoverMarkup(
-                synthetic
-              )
-        )
-      }
-
+    val interestingSynthetics = syntheticsAtLine.collect {
+      case (range, text)
+          if range.getEnd().getCharacter() == position.getCharacter() =>
+        text
+    }.distinct
+
+    if (interestingSynthetics.nonEmpty)
```

Comment:
Changed here to gather all synthetics at a spot - can happen especially with multiple implicit conversions at the same spot.